### ISSUE TITLE
Failed to run with the latest version

### ISSUE THREAD

**Issue by LiamDawe** [matanui159/ReplaySorcery#79]
**Describe the bug**
Trying to actually turn it on doesn't work in the latest build from the AUR.

**Desktop:**
 - Distro: EndeavorOS (Arch)
 - Desktop Environment: MATE
 - Window Manager: Marco
 - CPU: Intel i7 5960x
 - GPU: nvidia 1080

**Configuration:**

https://gist.github.com/LiamDawe/6dfd48647be9f9927cbc67df771d1473

**Logs:**
[replay-sorcery.log](https://github.com/matanui159/ReplaySorcery/files/5734583/replay-sorcery.log)

Looks like a missing option?

> Dec 23 10:15:26 liam-pc replay-sorcery[24144]: Setting 'width' to 'auto'...
> Dec 23 10:15:26 liam-pc replay-sorcery[24144]: Failed to set 'width' to 'auto': Option not found
> Dec 23 10:15:26 liam-pc replay-sorcery[24144]: Option not found

Seems to error out right after that.

**Comment by matanui159** [comment]
Do you still have `~/.config/replay-sorcery.conf` from the old version? The config options have changed alot

**Comment by LiamDawe** [comment]
I did actually, so I've removed that. I also copied the config file from here on GitHub in case it was an issue with the file provided with the AUR build.

Tried again, and it seems now something else says it's not found?

https://gist.github.com/LiamDawe/f3d50b7e001d5d0e542f0c07db3d1e73

[quote]Dec 23 10:31:24 liam-pc replay-sorcery[25765]: Setting 'videoInput' to 'auto'...
Dec 23 10:31:24 liam-pc replay-sorcery[25765]: Setting 'videoDevice' to 'auto'...
Dec 23 10:31:24 liam-pc replay-sorcery[25765]: Failed to set 'videoDevice' to 'auto': Option not found
Dec 23 10:31:24 liam-pc replay-sorcery[25765]: Option not found
Dec 23 10:31:24 liam-pc systemd[3691]: replay-sorcery.service: Main process exited, code=exited, status=1/FAILURE
Dec 23 10:31:24 liam-pc systemd[3691]: replay-sorcery.service: Failed with result 'exit-code'.[/quote]
